### ISSUE TITLE
Please Consider trying to merge apache-websocket in upstream Apache

### ISSUE THREAD

**Issue by clopez** [disconnect/apache-websocket#27]
Hi,

I don't know if you were already aware of this bug in upstream Apache: https://issues.apache.org/bugzilla/show_bug.cgi?id=47485

It would be nice if you were willing to make the effort of merging this nice module in upstream Apache.

Thanks

**Comment by jimjag** [comment]
FWIW, I would help in this process. I am currently adding 'ws' support in mod_proxy (to allow mod_proxy to simply proxy/forward ws requests) to trunk

**Comment by jej2003** [comment]
This is really great stuff.  Is the plan to also fwd wss requests as well?  That is the biggest issue that I am currently seeing.  Things work great over ws:// though, great job!

**Comment by jchampio** [comment]
I'm trying to pick up this effort with [my fork of mod_websocket](https://github.com/jchampio/apache-websocket).

**Comment by jimjag** [comment]
As mentioned, if there is interest in this being donated to the ASF, I can
help w/ that.

**Comment by jchampio** [comment]
What does the donation process look like? The module is still "v0" and the public API should not be considered stable at this point, but I'm definitely interested in cleaning it up to the point that the ASF can take it.

(If it requires copyright assignment, we're out of luck. But hopefully ASF doesn't require copyright assignment?)

**Comment by jimjag** [comment]
No, it doesn't require copyright assignment, simply a donation grant (
http://www.apache.org/licenses/software-grant.txt)
